ospfd/ospf_zebra.c

```diff
@@ -951,11 +1018,21 @@ static int ospf_zebra_read_route(int command, struct zclient *zclient,
 
 	ifindex = api.nexthops[0].ifindex;
 	nexthop = api.nexthops[0].gate.ipv4;
+	rt_type = api.type;
 
 	memcpy(&p, &api.prefix, sizeof(p));
 	if (IPV4_NET127(ntohl(p.prefix.s_addr)))
 		return 0;
 
+	/* Re-destributed route is default route.
+	 * Here, route type is used as 'ZEBRA_ROUTE_KERNEL' for
+	 * updating ex-info.But in resetting (no default-info
+	 * originate)ZEBRA_ROUTE_MAX  is used to delete the ex-info.
+	 * Resolved this inconsistency by maintaining same route type.
+	 */
+	if (is_prefix_default(&p))
```

Comment:
Please check typo in comment: space between 'ex-info.' and 'But', extra space between 'ZEBRA_ROUTE_MAX' and 'is used'